Application/Telegram/Keyboards/SendBookingListKeyboard.cs

```diff
@@ -5,13 +5,14 @@ namespace Application.Telegram.Keyboards
 {
     public static class SendBookingListKeyboard
     {
-        public static InlineKeyboardMarkup BuildKeyboard(List<BookingDto> bookings)
+        public static InlineKeyboardMarkup BuildKeyboard(List<BookingDto> bookings, string stateName="defaultState")
         {
             List<InlineKeyboardButton> buttons = new();
             foreach (var booking in bookings)
             {
                 buttons.Add(new InlineKeyboardButton($"Owner: {booking.UserName}") { CallbackData = booking.WorkplaceId.ToString() });
             }
+            buttons.Add(new InlineKeyboardButton($"◀️Go Back") { CallbackData = $"goBack|{stateName}" });
```

Comment:
one thing it can be moved to helper and use ass some extension method, that will be applied to list<InlineKeyboard>